jbmc/unit/java_bytecode/java_bytecode_parser/Outer.java

```diff
@@ -0,0 +1,7 @@
+class Outer {
+  class Inner {
+  }
+
+  Outer anonymous = new Outer() {
```

Comment:
:no_entry_sign: I don't understand the parsing code - please add a test that contains a class with more than one inner class inside it

Reply:
As discussed in person, `Outer.java` already compiles into 3 `.class` files, as the `Outer` class contains both an inner class and an anonymous class.